### PR TITLE
Fix color-dots aligment

### PR DESCRIPTION
**Pull request by gdostie**
- Fixed flashing color dots animation (was translating upwards while scaling making it unaligned)
- Added examples for color-dots

image

### CODE REVIEW

File: scss/animations.scss [hunk 1]
```diff
@@ -29,10 +29,10 @@
 
 @keyframes scaleoutCentered {
   0% {
-    transform: scale(0.0) translateY(-50%);
+    transform: scale(0.0);
   }
   100% {
-    transform: scale(1.0) translateY(-50%);
+    transform: scale(1.1);
```
Comment (sroy3) [inline scss/animations.scss]: Why did you add `.1`?

Comment (gdostie) [reply scss/animations.scss]: Since opacity is 0 at 100%, it appeared smaller than the non-flashing color dots. By adding .1 it seems the same size.

File: docs/_components/utility/04_color-dot.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+<p class="p1">Standard color dots</p>
+<i class="color-dot" style="height: 16px; width: 16px;"></i>
+<i class="color-dot state-critical" style="height: 16px; width: 16px;"></i>
+<i class="color-dot state-major" style="height: 16px; width: 16px;"></i>
+<i class="color-dot state-info" style="height: 16px; width: 16px;"></i>
+<i class="color-dot state-disabled" style="height: 16px; width: 16px;"></i>
+<i class="color-dot state-waiting" style="height: 16px; width: 16px;"></i>
+
+<p class="p1">Flashing color dots</p>
+<i class="color-dot state-executing" style="height: 16px; width: 16px;"></i>
```
Comment (vseguin) [inline docs/_components/utility/04_color-dot.html]: Since you always add those, should we add the height and width in the CSS directly?

File: scss/animations.scss (hunk 1 above)
Comment (vseguin) [reply scss/animations.scss]: I think it is worth a small comment as it is quite random at a glance.

File: docs/_components/utility/04_color-dot.html (hunk 2 above)
Comment (gdostie) [reply docs/_components/utility/04_color-dot.html]: I agree it would make sense, I'll release a minor and make sure this rule doesn't override any existing style in our other repos.
